pkgs/development/tools/mold/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, lib, autoPatchelfHook, cmake, tbb, llvmPackages_latest, xxHash, zlib, openssl }:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, lib, autoPatchelfHook, cmake, tbb, llvmPackages_latest, xxHash, zlib, openssl }:

stdenv.mkDerivation rec {
```

Reply:
The line is long enough. How about:

```nix
{
  stdenv,
  fetchFromGitHub,
  …
  openssl,
}:
```
?

Reply:
If you want but put the ``,`` at the beginning of the line.